Description: Added a trailing slash to the prefix
 .
 vanitygen (0.21-1~saucy8) saucy; urgency=low
 .
   * Initial packaging for Debian (Closes: #718860)
Author: Richard Ulrich <richi@paraeasy.ch>
Bug-Debian: http://bugs.debian.org/718860

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- vanitygen-0.21.orig/CMakeLists.txt
+++ vanitygen-0.21/CMakeLists.txt
@@ -59,7 +59,7 @@ IF(OPENCL_FOUND)
 	)
 	SET_TARGET_PROPERTIES(oclvanitygen PROPERTIES
 #	    VERSION
-        COMPILE_DEFINITIONS VG_OCL_SOURCE_PREFIX="${VG_OCL_SOURCE_PREFIX}"
+        COMPILE_DEFINITIONS VG_OCL_SOURCE_PREFIX="${VG_OCL_SOURCE_PREFIX}/"
 	)
 	INSTALL(TARGETS oclvanitygen RUNTIME DESTINATION bin)
 	INSTALL(FILES calc_addrs.cl DESTINATION share/vanitygen/scripts)
